graphdatascience/gds_session/schema.py

```diff
@@ -0,0 +1,12 @@
+from enum import Enum
```

Comment:
we should consider to add these as high-level classes (add to __init__.py). Not for this PR, but before promoting the feature

Reply:
what does that do? it seems like importing does still work

Reply:
instead of `from graphdatascience.session import Schema`instead of `from graphdatascience.session.schema import Schema`. Or even `from graphdatascience import Schema`

Reply:
aha